tests/testthat/test-combined_seasonal_output.R

```diff
@@ -1,5 +1,32 @@
-test_that("Output only includes one season", {
+test_that("Test that selection of current and all seasons work as expected", {
+  start_date <- as.Date("2021-01-04")
+  end_date <- as.Date("2023-12-31")
+
+  weekly_dates <- seq.Date(from = start_date,
+                           to = end_date,
+                           by = "week")
+
+  obs <- stats::rpois(length(weekly_dates), 1000)
+
+  tsd_data <- to_time_series(
+    observation = obs,
+    time = as.Date(weekly_dates),
+    time_interval = "week"
+  )
+
+  current_season <- epi_calendar(end_date)
+
+  current_season_output <- combined_seasonal_output(tsd_data, only_current_season = TRUE)
+  all_seasons_output <- combined_seasonal_output(tsd_data, only_current_season = FALSE)
+
+  expect_equal(unique(current_season_output$onset_output$season), current_season)
+  expect_equal(unique(current_season_output$burden_output$season), current_season)
+
+  expect_gt(length(unique(all_seasons_output$onset_output$season)), length(current_season))
```

Comment:
`length(current_season)` is just 1, right?

If yes, it is easier for me to read if you just put `expect_gt(..., 1)`

Reply:
yes length(current_season) is 1, bus expect_gt expects it to be greater than 1